plugins/BEdita/Core/tests/TestCase/ORM/Inheritance/QueryTest.php

```diff
@@ -219,6 +219,7 @@ public function aliasFieldProvider()
      * @dataProvider aliasFieldProvider
      * @covers ::fixAliasField()
      * @covers ::extractField()
+     * @covers ::aliasChecker()
```

Comment:
should we add `* @covers ::inheritedTables()`?

(it doesn't increment code coverage, but it's another method called by `fixAliasField()`)